imports/plugins/core/accounts/client/containers/auth.js

```diff
@@ -159,9 +155,7 @@ class AuthContainer extends Component {
     });
   }
 
-  hasPasswordService = () => {
-    return !!Package["accounts-password"];
-  }
+  hasPasswordService = () => !!Package["accounts-password"]
```

Comment:
Wasn't sure where we landed on the missing semicolons, are we just rolling without them for these kind of arrow func?

Reply:
@spencern @aldeed Where did we land on those semi-colons? Is this OK since it passes the linter?

Reply:
I'd prefer to have them, but not sure which rule needs to be enabled to turn them on. If we don't have a rule requiring/eliminating them, it seems we're likely to end up with inconsistencies from file to file.

Reply:
It looks to me like tslint and prettier both enforce semicolons here (class functions), but that eslint doesn't. I can't seem to find an option to enable them for eslint either.
https://palantir.github.io/tslint/rules/semicolon/
https://github.com/prettier/prettier/issues/1444

Reply:
@spencern I agree. Maybe don't worry about it for this issue, but you could create a ticket to pull in Prettier with just that rule enabled as a trial?